modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/identity.xml.j2

```diff
@@ -146,117 +146,190 @@
         <!-- Token cleanup feature config to clean IDN_OAUTH2_ACCESS_TOKEN table-->
         <TokenCleanup>
             <!--If true old access token cleaning feature is enabled -->
-            <EnableTokenCleanup>{{oauth.token.cleanup.enable}}</EnableTokenCleanup>
+            <EnableTokenCleanup>{{oauth.token_cleanup.enable}}</EnableTokenCleanup>
             <!--If true  old access token retained in audit table  -->
-            <RetainOldAccessToken>{{oauth.token.cleanup.backup_for_auditing}}</RetainOldAccessToken>
+            <RetainOldAccessToken>{{oauth.token_cleanup.retain_access_tokens_for_auditing}}</RetainOldAccessToken>
         </TokenCleanup>
         <!-- Specify the Token issuer class to be used.
              Default: org.wso2.carbon.identity.oauth2.token.OauthTokenIssuerImpl.
              Applicable values: org.wso2.carbon.identity.oauth2.token.JWTTokenIssuer
         -->
-        {% if oauth.token.token_generator_class is defined%}
-        <IdentityOAuthTokenGenerator>{{oauth.token.token_generator_class}}</IdentityOAuthTokenGenerator>
+        {% if oauth.extension.token_generator is defined%}
+        <IdentityOAuthTokenGenerator>{{oauth.extension.token_generator}}</IdentityOAuthTokenGenerator>
+        {% else %}
+        <IdentityOAuthTokenGenerator>org.wso2.carbon.identity.oauth2.token.OauthTokenIssuerImpl</IdentityOAuthTokenGenerator>
         {% endif %}
         <!-- True, if access token alias is stored in the database instead of access token.
              Eg. token alias and token is same when default AccessTokenValueGenerator is used.
              When JWTTokenIssuer is used, jti is used as the token alias
              Default: true.
              Applicable values: true,false
         -->
-        <PersistAccessTokenAlias>{{oauth.token.persist_access_token_alias}}</PersistAccessTokenAlias>
+        <!--TODO: Move to the interface of token generator -->
```

Comment:
Please create a GH issue and remove the TODO here. And have only a comment here that says, this will be moved to an interface method.

Reply:
Fixed. 
GH issue: https://github.com/wso2/product-is/issues/5145